packages/contracts-bedrock/scripts/Chains.sol

```diff
@@ -4,6 +4,7 @@ pragma solidity ^0.8.0;
 /// @notice Chain IDs for the various networks.
 library Chains {
     uint256 internal constant RootstockRegtest = 33;
+    uint256 internal constant OPRootstockRegtest = 42069;
```

Comment:
Out of curiosity: is this an arbitrary number?

Reply:
I don't know where it came from. I copied it from somewhere, I think Francisco used it first.

Reply:
cc @franciscotobar